src/server/sample_data/routes/lib/load_data.test.js

```diff
@@ -38,3 +38,23 @@ test('load log data', async () => {
   expect(myDocsCount).toBe(14005);
   expect(count).toBe(14005);
 });
+
+test('load ecommerce data', done => {
+  let myDocsCount = 0;
+  const bulkInsertMock = (docs) => {
+    myDocsCount += docs.length;
+  };
+<<<<<<< HEAD
```

Comment:
There are still merge conflicts in the test file that need to be fixed

Reply:
I believe I fixed this as well